src/scripts/cp.js

```diff
@@ -676,8 +678,10 @@ CoursePresentation.prototype.showKeywords = function () {
  * @param {number} value 0 - 100
  */
 CoursePresentation.prototype.setKeywordsOpacity = function (value) {
-  const [red, green, blue] = this.$keywordsWrapper.css('background-color').match(/\d+/g);
-  this.$keywordsWrapper.css('background-color', `rgba(${red}, ${green}, ${blue}, ${value / 100})`);
+  if (this.$keywordsWrapper.css('background-color') !== '') {
+     const [red, green, blue] = this.$keywordsWrapper.css('background-color').match(/\d+/g);
+     this.$keywordsWrapper.css('background-color', `rgba(${red}, ${green}, ${blue}, ${value / 100})`);
```

Comment:
@larsmagu Did you misconfigure your IDE? It looks like there's a 5 space indentation here.